tools/executors/esbuild/src/main.ts

```diff
@@ -35,12 +36,14 @@ export default async (options: EsbuildExecutorOptions, context: ExecutorContext)
   try {
     await readdir(options.outputPath);
     await rm(options.outputPath, { recursive: true });
-  } catch {}
+  } catch { }
 
   // TODO(wittjosiah): Workspace from context is deprecated.
   const packagePath = join(context.workspace!.projects[context.projectName!].root, 'package.json');
   const packageJson = JSON.parse(await readFile(packagePath, 'utf-8'));
 
+  const runtimeDeps = new Set([...Object.keys(packageJson.dependencies ?? {}), ...Object.keys(packageJson.peerDependencies ?? {})]);
```

Comment:
why do this outside the plugin? feels like something the plugin should know how to do given the resolveDir